app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionFeedbackViewModel.java

```diff
@@ -60,4 +66,29 @@ public void setSessionTitle(String sessionTitle) {
         this.sessionTitle = sessionTitle;
         notifyPropertyChanged(BR.sessionTitle);
     }
+
+    public void onClickSubmitFeedbackButton(@SuppressWarnings("unused") View view) {
+        if (callback != null) {
+            callback.onClickSubmitFeedback();
+        }
+    }
+
+    public void submitSessionFeedback(SessionFeedback sessionFeedback, @NonNull Callback callback) {
+        sessionsRepository.submitSessionFeedback(sessionFeedback).subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(s -> callback.onSubmitSuccess(), s -> callback.onSubmitFailure());
+    }
+
+    public void setCallback(@NonNull Callback callback) {
+        this.callback = callback;
+    }
+
+    public interface Callback {
+
+        void onClickSubmitFeedback();
+
+        void onSubmitSuccess();
```

Comment:
I think this `onClickSubmitFeedback` and `onSubmitSuccess` methods is not necessary if you call `.subscribe()` in `SessionFeedbackFragment`.